docs/PROVISIONING_OPENSTACK.md

```diff
@@ -0,0 +1,104 @@
+# OpenShift on OpenStack using CASL
```

Comment:
Why is this showing up as deleting one doc and adding new rather than just a rename/move?

Reply:
Good question and I don't know. I believe it's a git/github artifact. I did a `git mv` of the files and then made the changes to them. Let me see what I can do -- give me a few minutes

Reply:
@etsauer ok, done ... Github still doesn't show the move, but the log does - i.e.:
```
> git log --follow --summary docs/PROVISIONING_OPENSTACK.md
commit 618d296c3272b8d5d376379d8a791cc84d797bce
Author: Øystein Bedin <bedin@redhat.com>
Date:   Mon Nov 20 12:38:07 2017 -0500

    Updating README files ++


commit c2dd4c7f05d810fe583f7bb3af81e95f59b06618
Author: Øystein Bedin <bedin@redhat.com>
Date:   Mon Nov 20 12:36:32 2017 -0500

    Moving PROVISIONING_OPENSTACK.md to its new home

 rename PROVISIONING_OPENSTACK.md => docs/PROVISIONING_OPENSTACK.md (100%)

commit e5af1c6aed206b59efd6dff99c6ab462cb1c82c9
Author: Øystein Bedin <bedin@redhat.com>
Date:   Mon Nov 20 12:35:54 2017 -0500

    Bringing back original file of PROVISIONING_OPENSTACK

 create mode 100644 PROVISIONING_OPENSTACK.md

commit 23da46fb0f76a20af3dfdf87952c72b53ed3a9c6
Author: Øystein Bedin <bedin@redhat.com>
Date:   Sat Nov 18 18:36:03 2017 -0700

    Updating README files ++

 delete mode 100644 PROVISIONING_OPENSTACK.md

commit ffadb9343b3bd7b027291cff4958dc20a550261d
Author: Eric Sauer <etsauer@gmail.com>
Date:   Fri Nov 17 11:29:34 2017 -0500

    Rebasing openstack image on top of openshift-ansible image (#95)
    
    * Rebasing openstack image on top of openshift-ansible image
    
    * Restructuring start scripts, entrypoints, etc.
    
    * Need to precreate .ssh dir so it's writeable by user
    
    * Removing need for editing resolv.conf; restructing openstack instructions into separate README
    
    * Updating image readme
    
    * Updating to 45-1 build
    
    * Minor image tweaks

 copy README.md => PROVISIONING_OPENSTACK.md (50%)

commit 63d273c8d3d28994da247e04017807eb72683607
Author: Øystein Bedin <oybed@users.noreply.github.com>
Date:   Thu Nov 16 08:49:03 2017 -0700

    Inventory vars to set root size for AWS instances (#124)
    
    * Adding size option for root volume
    
    * Fixing typos and updating inventory with new values
    
    * Updating defaults


commit 1f1e13aad30879dccd00db458079735853865017
Author: Marcos Entenza <makentenza@gmail.com>
Date:   Mon Nov 13 03:59:52 2017 +0000

    New role to support provisioning in AWS (#106)
    
    * Several changes and new playbooks to support AWS deployment
    
    * Solve rebase issues from upstream master
    
    * Initial work to move to HA
    
    * Apply requested changes from PR's review
    
    * Revert "Initial work to move to HA"
    
    This reverts commit 2753dab8124e200f7c039c836524edfd0ef44c19.
    
    * Add env_id to ec2 tag filters to prevent name clashes
 :
 :
```